### PR TITLE
Feature/2 login

### LINKED ISSUE #2: Create Login Screen
# What we want :telescope:
Create a login screen that the user can feel it.

# Acceptance Criteria :sparkles: 
Able to insert the name and any password and when click at login go to dashboard page.

# T-Shirt :tshirt:

### PR DESCRIPTION
**Pull request by Schieck**
This pull request resolves #2 issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+
+## Running end-to-end tests
+
+Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/).
+
+## Further help
 
-### Front-End
-Where the app are rendered.
+To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI README](https://github.com/angular/angular-cli/blob/master/README.md).
```
Comment (Bankow) [inline README.md]: Revert these changes

Comment (Schieck) [reply README.md]: Adjusted in next commits
